clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -976,6 +976,27 @@ static QualType GetSYCLKernelObjectType(const FunctionDecl *KernelCaller) {
   return KernelParamTy.getUnqualifiedType();
 }
 
+static CXXMethodDecl *getOperatorParens(const CXXRecordDecl *Rec) {
+  for (auto *MD : Rec->methods()) {
+    if (MD->getOverloadedOperator() == OO_Call)
+      return MD;
+  }
+  return nullptr;
+}
+
+// Fetch the associated call operator of the kernel object
+// (of either the lambda or the function object).
+CXXMethodDecl *IsCallOperatorDefined(const CXXRecordDecl *KernelObjType) {
```

Comment:
Perhaps change the name to GetCallOperatorIfDefined or something?   It is no longer a bool function - so IsBlah does not seem right.